devel/ci/integration/tests/test_bodhi.py

```diff
@@ -16,6 +16,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
+import math
```

Comment:
Since ```math``` is part of the standard library, we should put a blank line between it and ```psycopg2``` for PEP-8.